examples/src/main/scala/org/apache/spark/examples/pythonconverters/HBaseConverters.scala

```diff
@@ -23,15 +23,27 @@ import org.apache.spark.api.python.Converter
 import org.apache.hadoop.hbase.client.{Put, Result}
 import org.apache.hadoop.hbase.io.ImmutableBytesWritable
 import org.apache.hadoop.hbase.util.Bytes
+import org.apache.hadoop.hbase.KeyValue.Type
+import org.apache.hadoop.hbase.CellUtil
 
 /**
- * Implementation of [[org.apache.spark.api.python.Converter]] that converts an
- * HBase Result to a String
+ * Implementation of [[org.apache.spark.api.python.Converter]] that converts all
+ * the records in an HBase Result to a String
  */
 class HBaseResultToStringConverter extends Converter[Any, String] {
   override def convert(obj: Any): String = {
+    import collection.JavaConverters._
     val result = obj.asInstanceOf[Result]
-    Bytes.toStringBinary(result.value())
+    val output = result.listCells.asScala.map(cell =>
+        "{'columnFamily':'%s','qualifier':'%s','timestamp':'%s','type':'%s','value':'%s'}".format(
```

Comment:
@davies 
Hi, Sorry for the delay.
Right now, for returning a string in json format is done.
However, I am thinking let the converter return a `Array[String]`, since there are several records for one row. Here what we are doing now is that we convert the `Buffer[String]` to `string` in the converter and then in python, we reconvert the 'string' to 'list'(or 'tuple'). 
If the converter return directly Array[String], there are two advantages:
1. We don't need to do conversion from 'string' to 'list' in python.
2. Right now, we should add a specific character to aggregate the `Buffer[String]` to a `String`(I am thinking "\n"). Therefore it is safer if we return `Array[String]`

How do you think about this? Thanks